hack/build/setup-pac-integration.sh

```diff
@@ -48,19 +52,21 @@ setup-pac-app() (
         payload="{ \"iss\": $PAC_GITHUB_APP_ID, \"iat\": ${now}, \"exp\": $((now+10)) }"
 
         webhook_secret=$(openssl rand -hex 20)
+        token=$(sign rs256 "$payload" "$(echo "$PAC_GITHUB_APP_PRIVATE_KEY" | base64 -d)")
+        # return  sprayproxy host in a loop until it is available
+        while ! kubectl get -n sprayproxy route/sprayproxy-route >/dev/null 2>&1 ; do
```

Comment:
@xinredhat As your manifest points to https://github.com/redhat-appstudio/sprayproxy/blob/main/config ,here I don't see [sprayproxy route](https://github.com/redhat-appstudio/sprayproxy/blob/main/config/route.yaml) inside `sprayproxy` namespace. May be that could be reason you are not getting the expected route

Reply:
@prietyc123 it's there, you can verify the route namespace by executing  `kubectl kustomize development` , it will print out all the final resources.  The follow is the route.
```
---
apiVersion: route.openshift.io/v1
kind: Route
metadata:
  annotations:
    haproxy.router.openshift.io/hsts_header: max-age=63072000
  name: sprayproxy-route
  namespace: sprayproxy
spec:
  port:
    targetPort: proxy
  tls:
    insecureEdgeTerminationPolicy: Redirect
    termination: reencrypt
  to:
    kind: Service
    name: sprayproxy
```

Reply:
Ahh my mistake